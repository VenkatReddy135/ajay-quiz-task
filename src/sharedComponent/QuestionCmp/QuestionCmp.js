import React from "react";

const QuestionCmp = (props) => {
  return (
    <div>
      <p className="fs-2">{props.content}</p>
    </div>
  );
};

export default QuestionCmp;
