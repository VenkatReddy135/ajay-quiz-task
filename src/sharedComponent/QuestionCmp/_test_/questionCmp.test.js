/* eslint-disable testing-library/no-node-access */
import React from "react";
import { cleanup } from "@testing-library/react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import QuestionCmp from "../QuestionCmp";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-17-updated";
Enzyme.configure({ adapter: new Adapter() });

describe("QuestionCmp  component", () => {
  const renderedData = renderer.create(<QuestionCmp />).toJSON();

  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<QuestionCmp />, div);
  });

  it("renders a snapshot", () => {
    expect(renderedData).toMatchSnapshot();
  });

  afterEach(() => {
    cleanup();
  });
});
