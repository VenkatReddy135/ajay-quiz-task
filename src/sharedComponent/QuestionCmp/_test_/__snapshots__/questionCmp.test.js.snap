// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionCmp  component renders a snapshot 1`] = `
<div>
  <p
    className="fs-2"
  />
</div>
`;
